api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -36,6 +36,7 @@
   private static final Object mutex = new Object();
 
   @Nullable private static volatile OpenTelemetry globalOpenTelemetry;
+  @Nullable private static Throwable setGlobalCaller;
```

Comment:
Add a annotation that this is guarded by the mutex to clarify synchronization story.